Formula/atlassian-cli.rb

```diff
@@ -1,20 +1,23 @@
 class AtlassianCli < Formula
   desc "Command-line interface clients for Atlassian products"
   homepage "https://bobswift.atlassian.net/wiki/pages/viewpage.action?pageId=1966101"
-  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-8.0.0-distribution.zip"
-  sha256 "cdba80bf44e8550ec9755c5b82711a0c5d32d2dbfe15bdbb55a75657603320b0"
+  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-8.2.0-distribution.zip"
+  sha256 "06604c908ab2e395e0f2d564081816d333d2025de54cea760150c565ed5d61a3"
 
   bottle :unneeded
 
   depends_on :java => "1.8+"
 
   def install
-    Dir.glob("*.sh") do |f|
-      cmd = File.basename(f, ".sh")
-      inreplace cmd + ".sh", "`dirname $0`", share
-      bin.install cmd + ".sh" => cmd
+    system "/usr/bin/unzip", "881099539"
```

Comment:
Hum, that's ugly. We should figure out what's really happening, instead.

Reply:
The upstream archive format has changed. The zip file now contains _another_ zip file, the naming scheme of which I have yet to figure out.

The current version of `atlassian-cli` also has undergone this change; currently `brew install atlassian-cli` will fail.

Reply:
Did it happen without checksum change? That would be impressive :)

Reply:
Indeed that would be. I wonder if maybe something has changed in the way we're unzipping stuff since the time 8.0.0 was merged in (September 1, 2018)?

Reply:
Yes, it may be a brew issue with unpacking. @reitermarkus is our expert there.

Reply:
I've reverted this as the issue should be fixed now that https://github.com/Homebrew/brew/issues/5895 is. FWIW I think would have been worth merging this as-is to unblock users who were unable to install this package.